nixie/unittests/test_render_diagram.py

```
@@ -39,24 +34,24 @@ async def fake_wait_for_proc(
-    expected = shlex.join(cmd)
+    mmd = tmp_path / "doc_1.mmd"
+    assert mmd.read_text() == block
+    svg = mmd.with_suffix(".svg")
+    expected = shlex.join(get_mmdc_cmd(mmd, svg, cfg_path))
     assert expected in caplog.text
 
 
 @pytest.mark.asyncio
-async def test_run_mermaid_cli_raises_on_failure(
+async def test_render_diagram_raises_on_failure(
     monkeypatch: pytest.MonkeyPatch, tmp_path: Path
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)